app/decorators/academic_year_decorator.rb

```diff
@@ -11,4 +11,13 @@ def display_name
       I18n.t("placements.academic_year.previous_academic_year", academic_year: academic_year.name)
     end
   end
+
+  def card_name
+    case academic_year
+    when AcademicYear.for_date(Date.current)
+      "#{academic_year.name} (current)"
```

Comment:
```suggestion
      "#{academic_year.name} #{I18n.t("placements.academic_year.current")}"
```

Could we add a translation here?